### PR TITLE
Allow building from git without GTK-doc installed

### PR DESCRIPTION
**Pull request by Thaodan**

### CODE REVIEW

**Comment by blivetghoul** [comment]
Can one of the admins verify this patch?

**Comment by Thaodan** [comment]
The patch works similar to how its done in gnutls see here:
https://gitlab.com/gnutls/gnutls/-/blob/master/bootstrap.conf#L80

File: autogen.sh [hunk 1]
```diff
@@ -15,13 +15,26 @@ cd "$srcdir"
 
 PKG_NAME=$(autoconf --trace 'AC_INIT:$1' configure.ac)
 
-gtkdocize --copy || exit 1
+enable_gtk_doc=--enable-gtk-doc
+gtkdocize || {
```
Comment (tbzatek) [inline autogen.sh:19]: Also this won't distinguish the scenario when the command is not available and actual failure from running it. Please add a separate check for the command availability.

**Comment by tbzatek** [comment]
Jenkins, ok to test.

**Comment by tbzatek** [comment]
No reply for almost two years, closing. Feel free to reopen with above mentioned fixes.

**Pull request closed without merging**